docs/Getting-Started.md

```diff
@@ -262,6 +262,13 @@ async function routes (fastify, options) {
     }
     return result
   })
+
+  fastify.post('/animals', async (request, reply) => {
+    // we can use the `request.body` object to get the data sent by the client.
+    // remember to validate the input data
```

Comment:
Input validation has not been covered at this point in the guide. This comment is therefore mysterious to the reader. Additionally, such examples should not be given. The example should include input validation, not omit it and put the burden on the reader to understand what is missing.

Reply:
I suggested this addition because we are inserting into mongo db the `request.body` object that is not secure.

do you think we should remove the sentence or improving it like:

```
remember to validate the input data, you will learn how to do it later in the Fastify documentation
```

Reply:
I think the example should really be added after validation has been covered. People _will_ copy and paste these examples into production apps. Leaving this example vulnerable will perpetuate the vulnerability to real world code.

Reply:
mmmm I think we can't protect people from themself.
Our duty here is writing the "step behind the yellow line" sign

Reply:
> mmmm I think we can't protect people from themself.

We can't ensure they do things correctly after the tutorial. What we _can_ do is control the information we present.